tests/README.md

```diff
@@ -0,0 +1,67 @@
+# HubSpot CLI E2E tests
+
+This project tests `hs.*` commands as if they were being used by an end-user.
+
+## Getting Started
+
+The main test execution is kicked off via the `run-tests` script. `run-tests` itself is a CLI, so to see the available options, you can `run-tests --help`. This was done so that the tests can be installed against any given version of the CLI and tested against it. In addition, it was done to keep the test logic separated from the CLI library.
+
+Note that if you are testing against a QA portal, not a PROD one, you'll need to add the `--qa` flag when running the script. There is still an outstanding issue with this because we attempt to add the `--qa` flag to all `hs` commands, however it is not available for all commands.
+
+### Configuration
+
+There are four ways to pass in necessary configuration to the script.
+
+1. Creating a .env file within the /tests folder.
+
+```bash
+PORTAL_ID="9289088"
+CLI_PATH="hs"
+PERSONAL_ACCESS_KEY="AiRiNGU2Y***************m1wLi2s8k2UlMYHEX"
+CLIENT_ID="123a4b56-****-****-****-************"
+CLIENT_SECRET="ab12345c-****-****-****-************"
+REFRESH_TOKEN="1234a5bc-****-****-****-************"
+```
+
+2. Through environment variables.
+
+```bash
+export PORTAL_ID="9289088"
+export CLI_PATH="hs"
+export PERSONAL_ACCESS_KEY="AiRiNGU2Y***************m1wLi2s8k2UlMYHEX"
+```
+
+3. Through arguments on the `run-tests` script
+
+```bash
+run-tests --portalId=9289088
+run-tests --cliPath=hs
+run-tests --personalAccessKey="AiRiNGU2Y***************m1wLi2s8k2UlMYHEX"
+```
+
+4. If you need to override any environment variables at a test variable, you can do that via the `setLocalTestOverrides` function available in `env.js`
+
+The priority is Local Test Overrides > CLI Arg Overrides > Environment Variables
+
+### Running Locally
+
+This repository should be added to the `hubspot-cli` repo, root folder, as a dependency. You can accomplish this locally by utilizing yarn linking.
+
+1. In `hubspot-cli-tests`, `yarn link`
+2. In `hubspot-cli`, `yarn link hubspot-cli-tests`
+3. To run it, `cd hubspot-cli`, `./node_modules/hubspot-cli-tests/run-tests`
```

Comment:
Can we add a script to the top-level `package.json` that is used to run the tests?